profiles/api/api.py

```diff
@@ -62,7 +63,10 @@ def _get(profile_id: str) -> Response:
         except ProfileNotFound as exception:
             raise NotFound(str(exception)) from exception
 
-        response = make_response(json.dumps(profile, default=normalize))
+        if request.headers.get('x-consumer-groups') == 'view-restricted-profiles':
```

Comment:
this header may have the form `view-restricted-profiles, view-something-else-unrelated, ...` so it needs to be parsed

Reply:
Not sure if case insensitivity (`X-Consumer-Groups` could be passed in) is handled by Flask for headers? It's part of the standard so I would expect so

Reply:
very good points. I'll update.

Reply:
Calling `get` on the `headers` object is case insensitive so i'll update the logic to handle the value which is returned in it's original case.